src/components/emoji-hud.js

```diff
@@ -180,24 +125,6 @@ AFRAME.registerComponent("emoji-hud", {
     this.el.sceneEl.removeEventListener("stateremoved", this._onThaw);
   },
 
-  tick() {
```

Comment:
>  I think we should also be shifting the focus of emoji behavior away from a flinging, or shaking behavior. I think emojis/reactions should largely take a passive role. You should spawn them (perhaps more than once), and then they shortly fade away. This minimizes abuse, accidental negative consequences of flinging an emoji, and provides consistent interactions across platforms.

I don't feel the same way. I think having the emojis be these juicy, silly objects we toss around has been a delightful addition and removing them will be our loss. I could definitely get behind something that addressed specific concerns about negative consequences (e.g. turn emojis transparent as they fly close to you so that it doesn't seem like they're hitting you in the head). Flinging them often indicates directional sentiment, though I'd like to see this expressed another way because I agree that it feels rude to fling emojis at someone (especially when they're unfamiliar with Hubs) in the current implementation.